packages/map-template/src/components/WebComponentWrappers/RouteInstructionsStep/RouteInstructionsStep.jsx

```diff
@@ -87,8 +93,10 @@ const RouteInstructionsStep = forwardRef(({ totalSteps, activeStep, previous, or
         }
     }, [substepsOpen]);
 
+
     return <mi-route-instructions-step
         ref={elementRef}
+        show-toggle-button={kioskLocation[0] === undefined || (kioskLocation[0] !== undefined && !isDesktop) ? true : false}
```

Comment:
We need that notation [0], because this is how the location object looks like 
![image](https://github.com/MapsPeople/web-ui/assets/113503166/e12482f2-f319-43ac-9c72-cc85b3d272b5)

Reply:
Removed the boolean condition ✅

Reply:
As we saw together, the [0] was a side effect of using `useRecoilState` instead of `useRecoilValue`. It has now been fixed.